frontend/pages/index.vue

```diff
@@ -41,16 +41,21 @@
                     <v-col cols="12" sm="3" md="3">
                       <v-text-field
                         v-model="collectionId"
-                        :rules="collectionRules"
+                        :rules="[v => !!v || 'Collection ID is required']"
+                        :items="collectionRules"
```

Comment:
rules should not replaced with items.
The rules for checking collection ID are checking two things:
the rule in line 44 and that collection is expressed in a `col` + number format.
See lines https://github.com/openstax/output-producer-service/blob/15acaca03d7da6a0bb3074e4d91deb252170c7f6/frontend/pages/index.vue#L221-L224

Reply:
@therealmarv All the fields have this construction
```
:rules="[v => !!v || 'Style is required']"
                        :items="styleItems"
```
I just copied them to each modasl field. Should I change them to the one you showed above? 
What is the .test in ```/^col\d*$/.test(v)```

Reply:
I think for this one remove the new added `:items` completely and leave line 44 the same as before:
```
:rules="collectionRules"
```

your test should still work. If you have any more question just ping me somewhere in Slack.